Sitenotice.php

```diff
@@ -10,19 +10,19 @@
 // and don't comment it out
 $wgMajorSiteNoticeID = 56;
 
-/* if ( !$wmgSiteNoticeOptOut ) {
+if ( !$wmgSiteNoticeOptOut ) {
 	$wgHooks['SiteNoticeAfter'][] = 'onSiteNoticeAfter'; // show to all users
 
 	function onSiteNoticeAfter( &$siteNotice, $skin ) {
 		global $wmgSiteNoticeOptOut, $snImportant;
 
 		$siteNotice .= <<<EOF
 			<table class="wikitable" style="text-align:center;"><tbody><tr>
-			<td style="font-size:125%">We are aware of issues where some wikis are unable to upload files, or are experiencing issues in thumbnail rendering. We are trying to resolve this as fast as we can, but we don't currently have an estimated time as to when this will be resolved. We apologise for the inconvenience this may cause, and thank you for your patience.</td>
+			<td style="font-size:125%">Miraheze will be doing some maintenace on our file storage. During this process images may be unavailable and you will not be able to upload images. Maintenance will last for 30 minutes and start at 20:00 UTC time on Thursday 20th.</td>
```

Comment:
- '_some_ maintenance'?
- This can be shortened to "Miraheze will be performing maintenance on May 17th, 2021 between 20:00 UTC and 20:30 UTC. During this maintenance you cannot view or upload images or videos."

Reply:
Thanks, done!